atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/TaskRunner.java

```diff
@@ -53,9 +53,7 @@ <V> List<V> runAllTasksCancelOnFailure(List<Callable<V>> tasks) {
 
         List<ListenableFuture<V>> futures = new ArrayList<>(tasks.size());
         for (Callable<V> task : tasks) {
-            DetachedSpan detachedSpan = DetachedSpan.start("task");
-            ListenableFuture<V> future = listeningExecutor.submit(task);
-            futures.add(attachDetachedSpanCompletion(detachedSpan, future, listeningExecutor));
+            futures.add(listeningExecutor.submit(task));
```

Comment:
Is the executor set up to attach spans inside properly? (the answer is yes if the `task` span already has child traces)

Reply:
No, as far as I can tell task has no children spans

Reply:
Might be worth double-checking that the listening executor is set up right (e.g. https://github.com/palantir/tracing-java/blob/develop/tracing/src/main/java/com/palantir/tracing/Tracers.java#L96)

Reply:
Talked about this offline and did some research after.

This was a problem before the code change.  The problem is since we are using executors on separate threads and how Tracing works, we lose connections to the main trace for anything in the executor.  Currently there are no spans under this executor, but that could in theory change and currently would not work correctly.

There are some solutions where we implement a new version of ListeningExecutorService to do the wrapping, but I am going to leave that out of this PR since it will balloon it.  I will implement a fix in a different PR and for this PR, just wrap the tasks I am touching.